client/client.js

```diff
@@ -99,15 +133,15 @@ function attachRegisteredObjectToLocalPlayerSynced(objectName, objectData) {
     }
 }
 
-function detachObjectFromLocalPlayerSynced() {
+function detachObjectFromLocalPlayerSynced(objectName) {
```

Comment:
The new parameter objectName has no function until this point, cause there is no handling at server side.
At this stage its only possible to hold one object at a time, therefor objectName doesn't need to be passed.
So this should be either removed again or implemented as a whole.